tests/rest/test_connection.py

```diff
@@ -2456,6 +2456,88 @@ def test_properties(self, con120):
             }
         }
 
+    def test_load_stac_from_job_canonical(self, con120, requests_mock):
+        requests_mock.get(
+            API_URL + "jobs/j0bi6/results",
+            json={
+                "links": [
+                    {
+                        "href": "https://wrong.link",
```

Comment:
FYI: .link is a valid top-level domain. In tests we preferably use TLDs that are reserved for testing (and will never be valid) e.g. [.test](https://en.wikipedia.org/wiki/.test)

Reply:
> TLDs that are reserved for testing (and will never be valid) e.g. [.test](https://en.wikipedia.org/wiki/.test)

I did not know this :open_mouth: